core/gather/driver/target-manager.js

```diff
@@ -104,19 +106,18 @@ class TargetManager extends ProtocolEventEmitter {
     const newSession = new ProtocolSession(cdpSession);
 
     try {
-      const target = await newSession.sendCommand('Target.getTargetInfo').catch(() => null);
-      const targetType = target?.targetInfo?.type;
-      const hasValidTargetType = targetType === 'page' || targetType === 'iframe';
+      const {targetInfo} = await newSession.sendCommand('Target.getTargetInfo');
```

Comment:
Before M110 this command would throw an error on worker targets. With this patch we would require a Chrome version >=M110 which is the current stable.